### PR TITLE
Deprecate MC_HOSTS_ env variable

### PR DESCRIPTION
**Pull request by kannappanr**
MC_HOSTS_ env variable will be replaced with MC_HOST_ as it is more
appropriate

MC_HOSTS_ will still be supported but will be removed in the near future.

### CODE REVIEW

**Comment by harshavardhana** [comment]
We should throw a warning message once MC_HOSTS_ is used @kannappanr

File: cmd/config.go [hunk 1]
```diff
@@ -275,7 +282,17 @@ func expandAlias(aliasedURL string) (alias string, urlStr string, hostCfg *hostC
 	// Extract alias from the URL.
 	alias, path := url2Alias(aliasedURL)
 
-	if envConfig, ok := os.LookupEnv(mcEnvHostsPrefix + alias); ok {
+	var envConfig string
+	var ok bool
+
+	if envConfig, ok = os.LookupEnv(mcEnvHostPrefix + alias); !ok {
+		envConfig, ok = os.LookupEnv(mcEnvHostsDeprecatedPrefix + alias)
```
Comment (ebozduman) [inline cmd/config.go]: What do we do if this `LookupEnv` returns `false`? 
If `false`, there is no point calling `expandAliasFromEnv` when `false`.
As I suggested in the above comment, we can return nil `hostCfg`

File: cmd/config.go [hunk 2]
```diff
@@ -178,7 +178,12 @@ func mustGetHostConfig(alias string) *hostConfigV9 {
 	// If alias is not found,
 	// look for it in the environment variable.
 	if hostCfg == nil {
-		if envConfig, ok := os.LookupEnv(mcEnvHostsPrefix + alias); ok {
+		if envConfig, ok := os.LookupEnv(mcEnvHostPrefix + alias); ok {
```
[the remark this answers is not in the document]
Comment (kannappanr) [reply cmd/config.go]: The second block will be removed soon as soon as decide to remove the deprecated environment variable. It is cleaner and easier to remove then.

[the remark this answers is not in the document]
Comment (ebozduman) [reply cmd/config.go]: OK

**Pull request merged**